src/devices/video/mc6845.h

```diff
@@ -125,6 +125,8 @@ class mc6845_device :   public device_t,
 	   updating is complete, end_update() */
 	uint32_t screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
 
+	void set_allow_interlace_video_mode(bool allow) { m_allow_interlace_video_mode = allow; };
+
```

Comment:
This should be grouped with the other configuration member functions.  Also, does it need to take the parameter at all?  It’s only ever going to be enabled at configuration time.